nvmeadm/tests/discovery_test.rs

```diff
@@ -156,7 +157,7 @@ fn connect_test() {
 
 #[test]
 fn disconnect_test() {
-    let _ = disconnect("mynqn");
+    disconnect("mynqn").expect("Should disconnect any existing devices");
```

Comment:
Just a nit related to the text message here: are we trying to disconnect any existing device or this specific disk/NQN ?
Shouldn't the message be something like "Should disconnect target device" or so ?

Reply:
Good point, done.